public/assets-sync.js

```diff
@@ -473,7 +508,7 @@ class ForgeAssetSync {
     async buildForgeInventory() {
         const forgeAssets = await ForgeAssetSync.getForgeAssets();
 
-        if (!forgeAssets) throw new Error("Could not error Forge VTT Assets Library content");;
+        if (!forgeAssets) throw new Error("Could not error Forge VTT Assets Library content");
```

Comment:
Looks like we've got a typo here. I think it should be this?
```suggestion
        if (!forgeAssets) throw new Error("Could not read Forge VTT Assets Library content");
```